docs/guides/4-custom-rulesets.md

```diff
@@ -402,38 +402,51 @@ Rulesets can then reference aliases in the [given](#given) keyword, either in fu
 
 Previously Spectral supported exceptions, which were limited in their ability to target particular rules on specific files or parts of files, or changing parts of a rule. Overrides is the much more powerful version of exceptions, with the ability to customize ruleset usage for different files and projects without having to duplicate any rules.
 
-Overrides can be used to:
+Overrides can be used to apply rulesets on:
 
-- Override rulesets to apply on particular files/folders `files: ['schemas/**/*.draft7.json']`
-- Override rulesets to apply on particular JSONPath's `files: ['**#/components/schemas/Item']`
-- Override rulesets to apply on particular formats `formats: [jsonSchemaDraft7]`
+- Particular formats `formats: [jsonSchemaDraft7]`
+- Particular files/folders `files: ['schemas/**/*.draft7.json']`
+- Particular elements of files `files: ['**#/components/schemas/Item']`
 - Override particular rules
 
 **Example**
 
+```yaml
+overrides:
+  formats:
+    - json-schema-draft7
+  files:
+    - schemas/**/*.draft7.json
+  rules:
+    valid-number-validation:
+      given:
+        - $..exclusiveMinimum
+        - $..exclusiveMaximum
+      then:
+        function: schema
+        functionOptions:
+          type: number
+```
+
+To apply an override to particular elements of files, combine a glob for a filepath
+with a [JSON Pointer](https://github.com/janl/node-jsonpointer) after the anchor, i.e.:
```

Comment:
I chose to point to the npm package used in Spectral for handling JSON pointers, but other options are to point to [RFC 6901](https://datatracker.ietf.org/doc/html/rfc6901) or a nice tutorial on JSON pointer (I couldn't find one).

Reply:
Yeah, that works I think, although we don't use that package directly anywhere IIRC.

Reply:
In that case I think linking to the RFC is probably better.

Reply:
Yeah, I like it. They should have some examples there too, so perhaps we link it to that section then?